storage/ballot_queue.go

```diff
@@ -393,12 +493,51 @@ func (s *Storage) MarkBallotBatchFailed(key []byte) error {
 		return fmt.Errorf("decode batch: %w", err)
 	}
 
-	// Mark all ballots in the batch as error
+	batchSize := len(agg.Ballots)
```

Comment:
this is declared here an it is used just a single time (warn trace) so I think that can be removed and use the value (`len(agg.Ballots`) in the log trace.